src/installation/guides/arm-devices/apple-silicon.md

```diff
@@ -0,0 +1,42 @@
+# Apple Silicon
+
+Void's Apple Silicon support is based on Asahi Linux. See their
+[website](https://asahilinux.org) and
+[documentation](https://github.com/AsahiLinux/docs/wiki) for more information
+about topics not covered here.
+
+## Installation
+
+Before installing, use the Asahi Linux install script to install "UEFI
+environment only". Then, [create a Live USB](../../live-images/prep.md) using an
+[Apple Silicon Void Linux ISO](https://voidlinux.org/download/#arm%20platforms).
+U-Boot (installed by the Asahi installer) should show the external USB as a boot
+option. If it does not, run these commands in the U-Boot prompt to boot:
+
+```
+setenv boot_targets "usb"
+setenv bootmeths "efi"
+boot
+```
+
+will instruct it to do so. Then proceed with the usual Void installation.
+
+To install, follow the [chroot install guide](../chroot.md), using the "XBPS
+method", observing the following modifications:
+
+For the base installation, install `base-system`, `asahi-base`, and
+`asahi-scripts`. This package provides important configurations and installs the
+necessary dependencies. When running `grub-install`, the `--removable` flag may
+be required. If using another bootloader, you may want to change the m1n1
+`PAYLOAD` in `/etc/m1n1.conf`.
```

Comment:
That should be `/etc/default/m1n1-kernel-hook`.
You could also add something like: 

> Edit `/etc/m1n1.conf` to change options for the m1n1 bootloader.